packages/lesswrong/components/forumEvents/ForumEventPoll.tsx

```diff
@@ -597,43 +614,41 @@ export const ForumEventPoll = ({
    */
   const clearVote = useCallback(
     async (e?: React.PointerEvent) => {
-      e?.stopPropagation();
-      setCurrentBucketIndex(DEFAULT_VOTE_INDEX);
-      setCurrentUserVote(null);
-      if (currentUser && event) {
-        setVoteCount((count) => count - 1);
-        await removeVote({ variables: { forumEventId: event._id } });
-        setCommentFormOpen(false);
-        refetch?.();
+      try {
+        if (currentUser && event) {
+          await removeVote({ variables: { forumEventId: event._id } });
+          setVoteCount((count) => count - 1);
+          setCommentFormOpen(false);
+          refetch?.();
+        }
+        e?.stopPropagation();
```

Comment:
Does `stopPropagation` actually work after an `await`? It might do - just seems a bit spooky.

Reply:
Well there's no need to find out 😌, I've moved this above